src/compile/header/index.ts

```diff
@@ -138,6 +159,11 @@ function getHeaderGroup(model: Model, channel: HeaderChannel, headerType: Header
 
       };
 
+      const update = {
+        ...defaults,
+        ...getHeaderLabelProperties(config)
```

Comment:
Note that title has a lot of native properties supported (see https://vega.github.io/vega/docs/title/). 

This means that in many cases you may not need to use `encode.update`

Reply:
From a quick skim, it looks like all title and label properties you're adding except (`x` and `y`) are supported by `title`. Thus, I wonder if it is better to map them directly to Vega `title` directive and drop supports for `x` and `y`. (They aren't very useful anyway.)

Reply:
Should we do this in a follow up or refactor this PR?

Reply:
This PR -- This is all new and should be easy to do.